app/ui/build.gradle

```diff
@@ -39,6 +39,7 @@ dependencies {
     implementation "net.jcip:jcip-annotations:1.0"
     implementation "com.jakewharton.timber:timber:${versions.timber}"
     implementation "org.apache.james:apache-mime4j-core:${versions.mime4j}"
+    implementation "org.apache.james:apache-mime4j-dom:${versions.mime4j}"
```

Comment:
This is really weird. Is it sufficient to put `testImplementation` here? Either way, please leave a comment describing why this is necessary

Reply:
I had a strange error on the testbench without it, I wrote that in the commit message. One of the tests (RecipentLoaderTest) could not find a class that is in org.apache.james:apache-mime4j-dom. Normally I would expect it to pass since the dependency is present (https://github.com/k9mail/k-9/blob/012befd59e08c43caddec47d42f5a5d73ac3c2f1/mail/common/build.gradle#L15).

I will try to reduce it to a testImplementation.